### ISSUE TITLE
Consider having Example site in /exampleSite/

### ISSUE THREAD

**Issue by rootkea** [adityatelange/hugo-PaperMod#301]
Hello!

https://themes.gohugo.io/hugo-papermod/ doesn't show "Demo" button. I think it's because it expects example site for demo to be in /exampleSite/

Ref: https://github.com/gohugoio/hugoThemes/blob/master/README.md#adding-a-theme-to-the-list

**Comment by adityatelange** [comment]
It's by choice to not show "Demo" button

**Comment by rootkea** [comment]
And what's the rationale behind that choice, if I may know?

**Comment by adityatelange** [comment]
> And what's the rationale behind that choice, if I may know?

I wish to host demo on GH pages ?

**Comment by rootkea** [comment]
Don't get me wrong. I like GH pages but why does hosting demo on GH pages require to make a choice to not show "Demo" button on themes.gohugo.io?

1. For many Hugo users themes.gohugo.io is a gateway to Hugo themes. Having a "Demo" button there helps with the visibility and adaption IMHO.

2. Also, when user git clones the theme repo, she receives `exampleSite/` directory which has config.toml set by theme author. For new users to theme that config.toml is very useful!

With the current set up when user clones the repo she doesn't receive `exampleSite` directory since the example site is in separate branch which is not clear or apparent to the beginner.

It's okay if you want to host demo on Github pages. Just copy that branch to `exampleSite/` directory so that above 2 issues will get resolved while continuing the demo on GH pages as you desire. :)

**Comment by kdkasad** [comment]
IMO, the example site should not be part of the theme tree. It wastes space for anyone using the theme, since the `exampleSite` directory will be there even if it isn't in use. The example site is also not really a part of the theme. It's more akin to a separate codebase, which is better represented by a separate orphan branch.

**Comment by rootkea** [comment]
> It wastes space for anyone using the theme, since the exampleSite directory will be there even if it isn't in use.

Now that I read this I agree with the current arrangement!

1. About having "Demo" button on themes.gohugo.io: There is a "Demo" link on https://themes.gohugo.io/hugo-papermod/ so I think we got this covered.

2. Regarding config.toml from exampleSite: Agagin, there is "ExampleSite can be found here: [exampleSite](https://github.com/adityatelange/hugo-PaperMod/tree/exampleSite)." on https://themes.gohugo.io/hugo-papermod/ linking the exampleSite source so this is covered too

Sorry for the noise! And thanks for the replies! :)

**Comment by adityatelange** [comment]
> > It wastes space for anyone using the theme, since the exampleSite directory will be there even if it isn't in use.
> 
> Now that I read this I agree with the current arrangement!
> 
> 1. About having "Demo" button on themes.gohugo.io: There is a "Demo" link on https://themes.gohugo.io/hugo-papermod/ so I think we got this covered.
> 2. Regarding config.toml from exampleSite: Again, there is "ExampleSite can be found here: [exampleSite](https://github.com/adityatelange/hugo-PaperMod/tree/exampleSite)." on https://themes.gohugo.io/hugo-papermod/ linking the exampleSite source so this is covered too
> 
> Sorry for the noise! And thanks for the replies! :)

So all these were already covered still you thought writing the reply below was a nice move ? 
Why would you do that and change your mind within an hour ?

> Don't get me wrong. I like GH pages but why does hosting demo on GH pages require to make a choice to not show "Demo" button on themes.gohugo.io?
> 
> 1. For many Hugo users themes.gohugo.io is a gateway to Hugo themes. Having a "Demo" button there helps with the visibility and adaption IMHO.
> 2. Also, when user git clones the theme repo, she receives `exampleSite/` directory which has config.toml set by theme author. For new users to theme that config.toml is very useful!
> 
> With the current set up when user clones the repo she doesn't receive `exampleSite` directory since the example site is in separate branch which is not clear or apparent to the beginner.
> 
> It's okay if you want to host demo on Github pages. Just copy that branch to `exampleSite/` directory so that above 2 issues will get resolved while continuing the demo on GH pages as you desire. :)

**Comment by rootkea** [comment]
> So all these were already covered still you thought writing the reply below was a nice move ?

I didn't look closely at the README on https://themes.gohugo.io/hugo-papermod/. Just noticed the missing "Demo" button among "Download" and "Homepage".

> Why would you do that

In addition to what's written above I wanted paperMod to have Demo button so that the theme can get more exposure with the new users So I opened this issue.

> and change your mind within an hour ?

As I said in my previous reply I changed my mind after reading @kdkasad's reply.

This issue could have been avoided if I had paid close attention to the README on https://themes.gohugo.io/hugo-papermod/ and thought about the redundant space used by `exampleSite/` directory.

Sue me!

But in all seriousness, I apologize for the noise again. :)
